tests/Cucumber/extra_testdata/good/extra_blank_lines_everywhere.feature

```diff
@@ -0,0 +1,47 @@
+
+# language:en
+
+@space-after-feature-tag
+
+Feature: Some feature
+
+    With a description that has
+
+    some blank lines before and after
+
+
+    Background: That is described
+
+        With multiple paragraphs
+
+        Of description
+
+        Given something
+
+        And something else
```

Comment:
This new feature file covers presence of blank lines between steps in a Background (and in all other parts of a gherkin file).